sdk/python/kfp/compiler/_component_builder.py

```diff
@@ -61,12 +61,10 @@ def download_gcs_blob(local_path, gcs_path):
 
 class DockerfileHelper(object):
   """ Dockerfile Helper generates a tarball with dockerfile, ready for docker build
-      arc_dockerfile_name: dockerfile filename that is stored in the tarball
-      gcs_path: the gcs path to store the tarball that contains both the dockerfile and the python file """
+      arc_dockerfile_name: dockerfile filename that is stored in the tarball """
```

Comment:
I'm not sure `arc_dockerfile_name` is needed. It's a name of a file you generate in a temporary archive. It does not even affect the resulting image in any way. Let's just name it `Dockerfile` and that's it.

Reply:
You are correct that the arc_dockerfile_name does not affect the resulting image. However, I think the DockerfileHelper class should provide the interfaces for different dockerfile names in case future scenarios require it. For example, we might want to create multiple dockerfiles inside a tarball such that we can launch multiple kaniko jobs based on the same tarball. We need the arc dockerfile name to be configurable to accommodate such scenarios.